inc/HttpClient/HttpClient.php

```diff
@@ -154,6 +160,7 @@ public static function retry_delay( int $retries, ?ResponseInterface $response )
 			}
 		}
 
+		// Convert it to milliseconds.
```

Comment:
I kept it this way so that both the calculated value, and the "retry-after" values are converted from s to ms.